src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java

```diff
@@ -79,7 +79,7 @@
  * @see RoutingAdviceTrait
  * @see ValidationAdviceTrait
  */
-public interface AdviceTrait {
+public interface AdviceTrait extends StatusConverter {
```

Comment:
Since you no longer depend on the `convert` method actually being a part of `AdviceTrait` one could argue that users of the library should be in charge whether to include this or not. Basically:

```java
@ControllerAdvice
public class ExceptionHandling implements ProblemHandling {
    // TODO
}
```

vs.


```java
@ControllerAdvice
public class ExceptionHandling implements ProblemHandling, StatusConverter {
    // TODO
}
```

What do you think?